EmployeesSalaries/EmployeesSalaries/Models/Interfaces/IReportsTo.cs

```diff
@@ -2,6 +2,8 @@
 {
     public interface IReportsTo
     {
-        IEmployee Manger { get; }
+        IEmployee Manager { get; }
```

Comment:
it's not always a manager, for example a dev is managed by a team lead, not a manager.
rename this property